CMakeLists.txt

```diff
@@ -104,6 +106,34 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# Qt
+#
+
+if(NOT CUKE_DISABLE_QT)
+    set(CMAKE_PREFIX_PATH $ENV{QTDIR})
+    find_package(Qt5Core  QUIET)
```

Comment:
Why do you search `QUIET` for these packages? I think it's rather helpful to have the output of these searches shown on the console.